app/routes/armoury/mission-board.tsx

```diff
@@ -23,25 +54,203 @@ export let loader = async ({ request }: LoaderArgs) => {
+											{loc(`loc_mission_name_${mission.map}`)}
+										</h2>
+										<div className="-mt-1 text-xs">
+											{loc(
+												`loc_zone_${
+													missionInfo[mission.map as keyof typeof missionInfo]
+														.zone_id
+												}`
+											)}
+										</div>
+
+										<p className="relative mt-2 text-xs invisible xs:visible">
+											{loc(
+												missionInfo[mission.map as keyof typeof missionInfo]
+													.mission_description
+											)}
+										</p>
+									</div>
+								</div>
+
+								<div className="absolute left-9 top-32 text-xs">
+									{mission.circumstance !== "default" ? (
+										<div className="relative pt-[0.375rem] align-text-middle text-sm text-yellow-400">
+											{loc(
+												circumstanceInfo[
+													mission.circumstance as keyof typeof circumstanceInfo
+												].display_name
+											)}
+											<div className="absolute w-10 h-10 -top-1 -left-12 border border-yellow-400 bg-gray-900 ">
+												<div
+													className="absolute w-[124px] h-[124px] top-0 left-0 bg-yellow-400"
+													style={{
+														WebkitMaskImage: `url(${img_url(
+															circumstanceInfo[
+																mission.circumstance as keyof typeof circumstanceInfo
+															].icon.replace("materials", "textures")
+														)})`,
+														maskImage: `url(${img_url(
+															circumstanceInfo[
+																mission.circumstance as keyof typeof circumstanceInfo
+															].icon.replace("materials", "textures")
+														)})`,
+														transformOrigin: "top left",
+														transform:
+															"scale(calc(40 / 124 * 0.8)) translate(10%, 10%)",
+													}}
+												></div>
+											</div>
+										</div>
+									) : (
+										<div className="relative h-[1.6rem]"></div>
+									)}
+
+									<div className="relative mt-5 text-sm">
+										<ul
+											className="mt-1 flex flex-row gap-6"
+											style={{
+												textShadow: "black 0 0 3px",
+											}}
+										>
+											<li>
+												<span
+													aria-label="Credits"
+													className="absolute w-16 h-16 top-1 bg-green-100"
+													style={{
+														WebkitMaskImage: `url(${img_url(
+															"glyphs/objective_credits"
+														)})`,
+														maskImage: `url(${img_url(
+															"glyphs/objective_credits"
+														)})`,
+														transformOrigin: "top left",
+														transform: "scale(calc(16 / 64))",
+													}}
+												/>
+												<span className="inline-block ml-6">
+													{mission.credits +
+														(mission.extraRewards?.circumstances?.credits || 0)}
+												</span>
+
+												{mission.extraRewards?.sideMission && (
+													<div className="relative ml-6 -mt-1 text-xs">
+														+ {mission.extraRewards.sideMission.credits}
+													</div>
+												)}
```

Comment:
```suggestion
												{mission.extraRewards?.sideMission ? (
													<div className="relative ml-6 -mt-1 text-xs">
														+ {mission.extraRewards.sideMission.credits}
													</div>
												) : null}
```

is my preferred style since this kind of code can lead to unintended output for numbers